CHANGELOG.md

```diff
@@ -38,6 +38,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     prevented AWS from halting/restarting warmed instances when task code was
     throwing consistent errors under load.
 
+### Added
+- **CUMULUS-2239**
+  - Add resource declaration to create a VPC endpoint in tea-map-cache module if `deploy_to_ngap` is false.
+- **CUMULUS-2063**
+  - Adds a new, optional query parameter to the `/collections[&getMMT=true]` and `/collections/active[&getMMT=true]` endpoints. When a user provides a value of `true` for `getMMT` in the query parameters, the endpoint will search CMR and update each collection's results with new key `MMTLink` containing a link to the MMT (Metadata Management Tool) if a CMR collection id is found.
+- **CUMULUS-2211**
+  - Adds `granules/bulkReingest` endpoint to `@cumulus/api`
```

Comment:
This whole section looks like a bad merge from upstream. These changelog notes should only be under "Unreleased". Can you remove these lines?